src/styles/_normolize.scss

```diff
@@ -0,0 +1,10 @@
+* {
```

Comment:
Consider renaming the file to '_normalize.scss' instead of '_normolize.scss' to follow the correct spelling.